python/mxnet/numpy_dispatch_protocol.py

```diff
@@ -168,6 +170,10 @@ def _register_array_function():
 _NUMPY_ARRAY_UFUNC_LIST = [
     'abs',
     'add',
+    #'equal',
```

Comment:
Please add these names after you fix the test as suggested in the comment below.